backend/src/graphql/resolver/CommunityResolver.test.ts

```diff
@@ -197,7 +221,9 @@ describe('CommunityResolver', () => {
               {
                 id: 6,
                 foreign: foreignCom3.foreign,
-                publicKey: expect.stringMatching('publicKey-ForeignCommunity'),
+                publicKey: expect.stringMatching(
+                  '4e3bf9536f694124c527b0aaf45aa6aea6c8c5d570d96b54f56f583724212b73',
+                ),
```

Comment:
Why not using the variable `foreignCom3.publicKey` instead of hardcoded string?

Reply:
FederatedCommunity.publicKey should be a Buffer type, graphql returns a hex string.
But you right I could make it way better. Save for future test, if the key will be actually validated.